### PR TITLE
Activate must-use plugins in the 'wp_loaded'-hook

### PR DESCRIPTION
**Pull request by tditlu**
This makes some poorly written woocommerce plugins work better...

### CODE REVIEW

**Comment by vinkla** [comment]
Great! Please look over the test suite before we can merge this.

File: src/PluginLoader.php [hunk 1]
```diff
@@ -214,7 +215,7 @@ protected function getActivePlugins(): array
      */
     protected function setActivePlugins(array $plugins): void
     {
-        sort($plugins);
+        $plugins = array_unique($plugins, SORT_REGULAR);
```
Comment (vinkla) [inline src/PluginLoader.php:217]: Isn’t SORT_REGULAR the default value?

**Comment by tditlu** [comment]
> Great! Please look over the test suite before we can merge this.

Done, also changed "preOptionActivePlugins", I know we dont want mixed variables, but blame it on Wordpress.

**Comment by vinkla** [comment]
Do you need this in a current project?

**Comment by tditlu** [comment]
> Do you need this in a current project?

Yes, but I will patch it with [cweagans/composer-patches](https://github.com/cweagans/composer-patches) until it's in some kind of release.

**Comment by vinkla** [comment]
Okay, great! Then we can release this in version 7.0!

File: src/PluginLoader.php (hunk 1 above)
Comment (tditlu) [reply src/PluginLoader.php:217]: True, 'SORT_REGULAR' can be omitted!

**Pull request merged**